### ISSUE TITLE
 WARNING  Failed to store DHT record for bag

### ISSUE THREAD

**Issue by igor9105** [xssnick/TON-Torrent#19]
Hi! 
I have tried download test torrent from `85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f` .
But Ton-Torrent can't find any info.
Just window  "Searching for torrent info..." and output from cmd below:
```
[STORAGE_DHT] CHECKING BAG OVERLAY FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] SEARCHING PEERS FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] DHT RETRY 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] SEARCHING PEERS FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
WARNING  Failed to store DHT record for bag 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f: failed to store value: zero copies made
```

**Comment by xssnick** [comment]
Hi, DHT should be fixed in [v1.1.0](https://github.com/xssnick/TON-Torrent/releases/tag/v1.1.0)

Please reopen if you will still have an issue

**Comment by igor9105** [comment]
After update.
Ton-Torrent can't find any info and  window still showing "Searching for torrent info...".
I had wait about 10 min.
```
[STORAGE] SEARCHING PEERS FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] DHT RETRY 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] SEARCHING PEERS FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] DHT RETRY 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] SEARCHING PEERS FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] DHT RETRY 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
[STORAGE] SEARCHING PEERS FOR 85d0998dcf325b6fee4f529d4dcf66fb253fc39c59687c82a0ef7fc96fed4c9f
```
